### PR TITLE
add land, lor parse function

### LINKED ISSUE #206: [parser] parse Land, Lor
**Subject**
parser

**Detail**
`&&`, `||` tokens are defined but there's no parsing function for those.

### PR DESCRIPTION
**Pull request by hea9549**
resolved: #206 

details:

add land, lor parse function

- [x] Test case

### CODE REVIEW

File: parse/parser.go [hunk 1]
```diff
@@ -38,6 +38,8 @@ var operatorMap = map[TokenType]ast.Operator{
 	GTE:      ast.GTE,
 	EQ:       ast.EQ,
 	NOT_EQ:   ast.NOT_EQ,
+	Land:	  ast.LAND,
```
Comment (boohyunsik) [inline parse/parser.go]: Should be LAND?

**Review by boohyunsik — CHANGES_REQUESTED** [review]

File: parse/parser.go (hunk 1 above)
Comment (hea9549) [reply parse/parser.go]: I make precedence name LAND and LOR.
So If I make Land to LAND, it has same name with precedence.
Also, other token name has lowercase like Slash, Mod, IntType and so on.
I think it is not bad to make name 'Land'.

File: parse/parser.go [hunk 2]
```diff
@@ -59,6 +61,8 @@ type precedence int
 const (
 	_ precedence = iota
 	LOWEST
+	LOR         // ||
+	LAND        // &&
```
Comment (zeroFruit) [inline parse/parser.go:65]: I think LOR and LAND should have same precedence. plz add test case which contain both `||` and `&&`

for example
```
true || true && true // should ((true || true) && true)
```
```
true && true || true // should ((true && true) || true)
```

Comment (hea9549) [reply parse/parser.go:65]: complete > <

Comment (zeroFruit) [reply parse/parser.go:65]: my mistake 

https://golang.org/src/go/token/token.go

**Review by zeroFruit — APPROVED** [review]

**Pull request merged**